### DISCUSSION TITLE
Expo Session already exisits

### DISCUSSION THREAD

**Discussion by Flowko** [clerk/javascript#700] [Support request - DO NOT OPEN HERE]
i cant seem to find a solution to this issue, so i have an expo app that i managed to set some of the main functionalities (sign-in/sign-up) with no issues, and i also have a next instance running, but when i send a reset password ( in this case an email_link email ) to the user from my expo app, to be able to reset his password, it loads fine in the next app and creates the session, but in my expo app, if i try to login i get `Session already exists`, which its correct since the session is already created through next, but is there a way to bypass this ? 

( note: when the user requests a reset password email, i redirect them to the login screen, since the email will open the link in my next app, this is another issue, my initial idea was to keep it in my expo app to let the user reset their password from it, but i couldn't get it to work, reading this https://clerk.dev/docs/authentication/forgot-password confuses me a bit, cus it was sending two emails, if you can help me with that as well that would be great, like what kinda of code would be used in both reset password and forgot password screens)

Thanks

**Comment by jescalan** [comment]
Sorry for the super late followup, but if this is still relevant, it would be wonderful if you could [open a bug report](https://github.com/clerkinc/javascript/issues/new?assignees=&labels=needs-triage&projects=&template=BUG_REPORT.yml) for this!
